dataset/dictionary/characters.json5

```
@@ -4364,6 +4378,51 @@
     zhCN: "纳齐森科鲁兹",
     tags: [ "fontaine", "character-sub" ],
   },
+  {
+    en: "Peruere",
+    ja: "ペルヴェーレ",
+    zhCN: "佩露薇利",
+    tags: [ "fontaine", "character-sub" ],
+    notes: "「アルレッキーノ」の本名",
```

IMO, Peruere should be considered a main character.
I also want to add `enemy-boss`.

```suggestion
    tags: [ "snezhnaya", "fontaine", "character-main", "enemy-boss" ],
    notes: "アルレッキーノの本名",
```